### PR TITLE
Basic path fixes and `app_label` changes

### PR DESCRIPTION
**Pull request by hiteshgarg14**

### CODE REVIEW

**Comment by hiteshgarg14** [comment]
@jamesturk 
This PR is ready for review.

File: opencivicdata/core/admin/views.py [hunk 1]
```diff
@@ -23,14 +23,14 @@ def merge_tool(request):
 
         diff = compute_diff(person1, person2)
 
-        return render(request, 'opencivicdata/admin/merge.html',
+        return render(request, 'opencividata/admin/merge.html',
```
Comment (jamesturk) [inline opencivicdata/core/admin/views.py]: oops, this is a typo in the directory, mind renaming the directory instead?

**Comment by jamesturk** [comment]
that directory name should be opencivicdata, my mistake!

**Comment by hiteshgarg14** [comment]
actually, the directory was renamed for core and legislative apps both. so I got confused. :+1:

**Pull request merged**